tests/validation/tests/v3_api/common.py

```diff
@@ -2366,6 +2367,9 @@ def receiver(self, socket, skip):
             except websocket.WebSocketConnectionClosedException:
                 print("Connection closed")
                 break
+            except websocket.WebSocketProtocolException as wpe:
```

Comment:
This is catching an exception in the Jenkins output -- seems that it's still trying to call `socket.recv()` after the connection has been closed: try `while True && socket.connected`

Reply:
I added the validation the `OSError` is not always reproducible but I've added the `socket.connected` as suggested.